docs/guides/migrating-from-camunda-platform-7/migration-readiness.md

```diff
@@ -0,0 +1,278 @@
+---
+id: migration-readiness
+title: Getting ready to migrate
+description: "Learn readiness indicators for migrating from Camunda Platform 7 to Camunda Platform 8."
+---
+
+:::note
+Migration of existing projects to Camunda Platform 8 is optional. Camunda Platform 7 still has ongoing [support](https://docs.camunda.org/enterprise/announcement/).
+:::
+
+## Migration overview
+
+Let's discuss if you need to migrate before diving into the necessary steps and what tools can help you achieve the migration.
+
+## When to migrate?
+
+New projects should typically be started using Camunda Platform 8.
+
+Existing solutions using Camunda Platform 7 might simply keep running on Camunda Platform 7. The platform has ongoing [support](https://docs.camunda.org/enterprise/announcement/), so there is no need to rush on a migration project.
+
+You should consider migrating existing Camunda Platform 7 solutions if:
+
+- You are looking to leverage a SaaS offering (e.g. to reduce the effort for hardware or infrastructure setup and maintenance).
+- You are in need of performance at scale and/or improved resilience.
+- You are in need of certain features that can only be found in Camunda Platform 8 (e.g. [BPMN message buffering](/components/concepts/messages.md#message-buffering), big [multi-instance constructs](/components/modeler/bpmn/multi-instance/multi-instance.md), the new Connectors framework, or the improved collaboration features in Web Modeler).
+
+## Migration steps
+
+For migration, examine development artifacts (BPMN models and application code), and workflow engine data (runtime and history) in case you migrate a process solution running in production.
+
+The typical steps are:
```

Comment:
Are there atypical steps, or other ways to do this?

Reply:
Not necessarily, but the order is not linear and some steps may need to be skipped or iterated on. Do you have a better way to phrase this?

Reply:
Is it worth noting that here, or should most users of the page recognize that?

Reply:
We should be ok, given the sentence below the list mentions this. If we get feedback there is confusion, we can rework this.